R/utils.R

```diff
@@ -44,3 +44,9 @@ collapse <- function(vector, collapse = ", ", limit = 150, end = "...") {
   out <- paste(vector, collapse = collapse)
   out
 }
+
+`%||%` <- function(a, b) if (is.null(a)) b else a
```

Comment:
could you reformat this (this is my fault)

Reply:
Yeah sorry, I had said I would do that and then subsequently forgot about it. Tidied up now.